### ISSUE TITLE
mode to exclude offensive words?

### ISSUE THREAD

**Issue by leffel** [openlogic/random-word#9]
Hi there! 👋

I wanted to write a quick issue to let you know about a situation we experienced a couple weeks ago!

We used random-word to generate passwords for a few dozen new user accounts for our web app. For security, we didn't have any visibility into the passwords being generated before sending them via email to the account holders. A couple days later, one of our users emailed us about a password they received that had an extremely upsetting word in it. It was the n-word. 😖

Ultimately, this isn't a problem with the gem at all - I know there's an exclusion list, probably exactly for this reason! However, we definitely didn't think about that ahead of time, and I imagine other folks may be in the same boat.

Anyways, just a heads up in case you were ever considering adding a mode that automatically excludes a predefined list of words, like [this one.](https://github.com/dariusk/wordfilter/blob/master/lib/badwords.json)

Thank you!

**Comment by toddthomas** [comment]
Hi @leffel. I'm really sorry to hear that. I use this gem for generating test data. I've spent many years watching tests run and seeing strings produced via the gem, and I've never seen that word before. I didn't know it was in the noun list. I regret that a thing I've participated in creating unintentionally upset someone.

I've heard from another contributor who may have a solution for issue. In the meantime, I'm happy to review a pull request from you or anyone else that may also have a solution to propose.
